business/definitionService.js

```diff
@@ -72,7 +72,14 @@ class DefinitionService {
     if (get(existing, '_meta.schemaVersion') === currentSchema) {
       this.logger.info('computed definition available', { coordinates: coordinates.toString() })
       result = existing
-    } else result = await this.computeAndStore(coordinates)
+    } else {
+      try {
+        result = await this.computeAndStore(coordinates)
+        return this._trimDefinition(this._cast(result), expand)
+      } catch (err) {
+        throw new Error(err.message)
```

Comment:
Does this do anything other than recreate the original Error? Not sure if the purpose is to hide the underlying call stack by overwriting from this point.

Reply:
That's a good point, let me experiment a bit

Reply:
Thank you! I was making this considerably more complicated than it had to be. I've fixed it up!